less/mixins.less

```diff
@@ -1 +1,6 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
+.flexcode{
+    display: flex;
+    font-size: 1.6rem;
+    justify-content: space-around;
```

Comment:
i'd like to see you create more mixins for reusability, but you did a great job with this project